source/core/output_buffers/ut_output_buffer_tmp.sql

```diff
@@ -1,4 +1,9 @@
-create table ut_output_buffer_tmp$(
+declare
+  v_table_sql varchar2(32767);
+  e_non_assm exception;
+  pragma exception_init(e_non_assm, -43853);
+begin
+  v_table_sql := 'create table ut_output_buffer_tmp$(
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: An illegal character with code point 10 was found in this literal. ([plsql:LiteralsNonPrintableCharactersCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3ALiteralsNonPrintableCharactersCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=816&issues=AWhCR0RdSi5GWQS9iTtd&open=AWhCR0RdSi5GWQS9iTtd)

[ISSUE_KEY]: AWhCR0RdSi5GWQS9iTtd